hack/verify-build.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+if [ -n "$(cluster/virtctl.sh version | grep dirty)" ]; then
```

Comment:
without a cluster being deployed, this is going to have a non-zero exit status, right?  

it woudl still convey the client's version correctly even without access to a cluster though.

Reply:
Yes. I also have to add a positive check (that "clean" appears).